### ISSUE TITLE
Castr.io

### ISSUE THREAD

**Issue by jeffreylsoffer** [bitfocus/companion-module-requests#340]
**Describe the feature**
Castr.io is used to restream RTMP and SRT feeds to various destinations (fb, yt, twitch, etc)
Ideally this module would be able to start and stop these feeds from Companion with feedback for if they're enabled.

**Is this platform dependent (windows, mac, ..)?**
It's web-based.

**If documentation is required to implement, do you know where to find it?**
https://castr.io/api

**Usecases**
In its most basic form, the module should get all the stream channels and destinations.
A button could then be assigned to the stream destinations (fb, yt, twitch, etc) and be able to start their feeds from Companion. 

Right now I use vMix to send to Castr.io and by doing this I'm giving up the feedback from the vmix module that starts external feeds, so I'm a bit unaware if my stream is actually going live without checking the castr.io dashboard.

**Comment by LCCmedia** [comment]
I would also like to see this added! Start/stop or toggle the destinations.

**Comment by map740** [comment]
I am also interested in this...

**Comment by sms42pdx** [comment]
Yes, I would also really like to see this feature added.  This would eliminate the need for a computer other than the companion pi for my live stream work flow.  I would be fine with a toggle all button for castr.io, but buttons for individual platforms would be nice also.  As castr does have an API I would think this should be possible.

I would also want to see feedback on the button showing that it indeed worked correctly with castr.io

**Comment by m1tk4** [comment]
this is now available in https://github.com/bitfocus/companion-module-castr-api
